drafting/conda-template.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+
+# This script reads in a yaml file (qiime2 recipe)
+# Pulls the needed data from that file
+# And uses that data to write to a secondary yaml file
+# Which will be the end result: a conda recipe
+
+import yaml
+import os
+import sys
+from jinja2 import Template
+
+if __name__ == '__main__':
+    # Reading the qiime2 recipe file
+    try:
+        qiime2_recipe = sys.argv[1]
+    except:
+        raise ValueError('Missing required parameter. QIIME 2 recipe filepath must be provided.')
+    
+    with open(qiime2_recipe) as qiime2_recipe:
+        parsed_recipe = yaml.load(qiime2_recipe, Loader=yaml.FullLoader)
```

Comment:
I would suggest choosing a new name for the second time you assign to `qiime2_recipe` - that kind of [shadowing](https://en.wikipedia.org/wiki/Variable_shadowing) can lead to pain later, in my experience. Maybe you could do something like 

```python
    with open(qiime2_recipe) as file_handle:
        parsed_recipe = yaml.load(file_handle, Loader=yaml.FullLoader)
```

We also often use `fh` as shorthand for `file_handle`, so that could work well here, too.